cips/cip-sampling-window.md

```diff
@@ -0,0 +1,38 @@
+| cip | XX (assigned by Editors) |
+| - | - |
+| title | Lower Sampling and Pruning Window to 14 days |
+| description | Lower Sampling and Pruning Window to 14 days |
+| author | Callum Waters ([@cmwaters](https://github.com/cmwaters)), Nashqueue ([@Nashqueue](https://github.com/Nashqueue)) |
+| discussions-to | URL |
+| status | Draft |
+| type | Standards Track |
+| category | Data Availability |
+| created | 2025-03-17 |
+| supercedes | [CIP 4](./cip-4.md) |
+
+## Abstract
+
+Lower the minimum sampling and pruning window to 14 days and 14 days + 1 hour respectively to reduce storage costs in response to increasing throughput of the network and to ensure that the subjective intialized header, known as **Tail**, in the header pruning CIP, is within the trusting period and thus not subject to long range attacks.
+
+## Parameters
+
+| Parameter     | Current value | Proposed value | Description  |
+|---------------|---------------|----------------|--------------|
+| RequestWindow | 720 hours  | 336 hours | This is analagous to the sampling window  |
+| StorageWindow | 721 hours | 337 hours | This is analagous to the pruning window - the minimum retention period of shares for serving the network |
+
+## Rationale
+
+The next pending increase in throughput at 32 MB blocks in 6 seconds could require as much as 1 TB of storage per day, at least 30TB of storage for pruned bridge nodes. With this CIP, it is reduced to 14TB making it less costly to serve data to the network. This may also promote the introduction of archival storage services or namespace pinning.
```

Comment:
Kinda difficult to follow this logic. Here's an optional re-wording

```suggestion
The next pending increase in throughput (32 MB blocks every 6 seconds) would require bridge nodes to store up to 1 TB per day. The current sampling window (30 days) implies that pruned bridge nodes need to have 30 TB of storage. This CIP reduces the sampling window to 14 days to reduce the storage requirement to 14 TB which makes it less costly to serve data to the network. 
```

Also this seems unrelated to the original rationale and can be a separate paragraph:

> This may also promote the introduction of archival storage services or namespace pinning.